site/en/rules/bzl-style.md

```diff
@@ -141,11 +141,14 @@ not instantiated by that macro), follow these best practices:
 
 *   A macro should take a `name` argument and define a target with that name.
     That target becomes that macro's *main target*.
-*   All other targets defined by a macro should have their names preceded with
-    an underscore (`_`), followed by the name attribute. For instance, if the
-    macro is supplied with the name *resources*, internal targets should have
-    names beginning with *_resources*. They should also have restricted
-    visibility.
+*   All other targets defined by a macro should:
+    *   Have their names preceded with an underscore (`_`), followed by the
+        value of the `name` attribute. For instance, if the macro is supplied
+        with the name *resources*, internal targets should have names
+        beginning with *_resources*.
+    *   Have restricted visibility (`//visibility:private`), and
```

Comment:
It's possible that a macro could generate an internal target for use by the same macro called in other packages. For instance, the macro could take `name="foo"` and generate "_foo_helper". Then in another package, it could take in `deps=["//pkg:foo"]` and munge that string into "//pkg:_foo_helper" to get the private target. In that case, the helper can't have private visibility.

But I don't think that's necessarily a style we should encourage, and anyone clever enough to do that is also clever enough to realize this advice doesn't apply to them anyway. So the current wording is fine.